### DISCUSSION TITLE
Scraping failed:  401

### DISCUSSION THREAD

**Discussion by trevtravtrev** [tweepy/tweepy#2110] [Questions]
Is anyone getting this message when trying to use get endpoints today?

**Comment by Harmon758** [comment]
This isn't reproducible or actionable at all, and there's no information with which to help you.
You need to provide what exactly you're trying and encountering, i.e. a reproducible example (MCVE/SSCCE) or your relevant code and the full traceback.

**Comment by SaranBodduluri** [comment]
Is tweepy working after the recent policy changes that were made in Feb 2023?

**Comment by Harmon758** [comment]
I'm not sure how exactly that relates to this issue or what specific policy changes you're referring to, but yes, Tweepy is functional.
